### ISSUE TITLE
No "celery inspect" support for periodic (or queued) tasks

### ISSUE THREAD

**Issue by ShaheedHaque** [celery/celery#4535]
## Checklist

- [N, Celery 4.1.0] I have included the output of ``celery -A proj report`` in the issue.
      (if you are not able to do this, then at least specify the Celery
       version affected).
- [Y] I have verified that the issue exists against the `master` branch of Celery.

## Steps to reproduce

I [struggled for some time](https://github.com/celery/django-celery-beat/issues/126) thinking that 'inspect scheduled' should show me the periodic tasks I had created using Beat, until I finally realised that 'scheduled' is not the same as 'periodic'. Along the way, I stumbled upon the related point that there is no supported way to list queued tasks either (though the discussion in kombu issue 422 leads to some interesting [code in Flower](https://github.com/mher/flower/blob/master/flower/utils/broker.py#L135)).

## Expected behavior

There should be some supported way to query the periodic (or queued) tasks in the system.

## Actual behavior

There is no supported way to list the periodic (or queued) tasks.

**Comment by auvipy** [comment]
whats your latest thought about the issue?

**Comment by ShaheedHaque** [comment]
I've moved away from using Celery partly because of the reason for this thread, and because my tasks were actually more one-off than strictly periodic (though I noticed that there seems to be some developments in the area which might be attractive).

The problem for me is that I feel that not having a way to interrogate the current actual state of the system in a meaningful way (and possibly modifying, or at least clearing such state) is a big deterrent from a system maintainability and diagnosability point of view. Being able to *see* what state is in RabbitMQ from a Beat perspective seems essential to me. 

To be honest, I'm even a bit wary of using baseline Celery for this reason, but at least there I can arrange that all delays are handled in my code, with only a "schedule immediate out of Django context" left to Celery, thus limiting the impact of the opaque state.
